examples/tutorials/Monomers_Graphs_Vectors.ipynb

```diff
@@ -0,0 +1,2735 @@
+{
```

Comment:
<p>Add these images in the text section, not as code blocks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4035/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>